src/scripts/main.js

```diff
@@ -357,4 +357,52 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+function calculateAge(born, died) {
+  return died - born;
+}
+
+function calculateCentury(died) {
+  return Math.ceil(died / 100);
+}
+
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('th');
```

Comment:
```suggestion
  const nameCell = document.createElement('td');
```
`th` is for table head